### PR TITLE
Update terms.md

### PR DESCRIPTION
**Pull request by amirmehrkar**
changes of content

### CODE REVIEW

File: snippets/terms.md [hunk 1]
```diff
@@ -3,27 +3,40 @@
...
+- [Acknowledgement and Sharing/Publication Policy](https://www.opensafely.org/policies-for-researchers/#acknowledgment-and-data-sharing--publication-policy)
+- [Information Governance and Ethics content Policy](https://www.opensafely.org/policies-for-researchers/#information-governance-and-ethics-content-policy).
 
-* all study definition code
-* codelists
-* results released from the platform (with disclosure controls applied)
+NNHS England ([the Data Controller for OpenSAFELY](https://www.england.nhs.uk/contact-us/privacy-notice/how-we-use-your-information/covid-19-response/coronavirus-covid-19-research-platform/)) oversees the final approval for all publication ready papers, reports or presentations, principally to check that the outputs align with the stated application purpose; NHS England has been extremely supportive of all research and analyses to date. The usual response time for approval is 1-2 weeks. There are other datasets we hold ([such as ICNARC, ISARIC, ONS-CIS](https://www.opensafely.org/policies-for-researchers/#for-the-datasets-listed-below)), which require additional approvals from their source institutions.
```
Comment (ghickman) [inline snippets/terms.md:14]: NNHS?

File: snippets/terms.md [hunk 2]
```diff
@@ -3,27 +3,40 @@
...
 
-* all study definition code
-* codelists
-* results released from the platform (with disclosure controls applied)
+NNHS England ([the Data Controller for OpenSAFELY](https://www.england.nhs.uk/contact-us/privacy-notice/how-we-use-your-information/covid-19-response/coronavirus-covid-19-research-platform/)) oversees the final approval for all publication ready papers, reports or presentations, principally to check that the outputs align with the stated application purpose; NHS England has been extremely supportive of all research and analyses to date. The usual response time for approval is 1-2 weeks. There are other datasets we hold ([such as ICNARC, ISARIC, ONS-CIS](https://www.opensafely.org/policies-for-researchers/#for-the-datasets-listed-below)), which require additional approvals from their source institutions.  
+
+Following approval, as part of the [principles you are signing up to](https://www.opensafely.org/principles/, at or before the time when your results are published (as  manuscripts, official reports and presentations), we expect you to make your project's GitHub repository public in order to share:
```
Comment (ghickman) [inline snippets/terms.md:16]: Missing a closing parenthesis at the end of the link URL.

File: snippets/terms.md [hunk 3]
```diff
@@ -3,27 +3,40 @@
...
 
 ## Your data in public OpenSAFELY dashboards
 In addition, OpenSAFELY will create a public dashboard listing for all approved studies:
 
 * description of purpose
-* contact email of researcher
+* contact email of study lead(s)
 * affiliated organisation
-* when first code run
+* when first code run against the patient data
```
Comment (ghickman) [inline snippets/terms.md:30]: "when code is first run"?

**Review by ghickman — CHANGES_REQUESTED** [review]

File: snippets/terms.md (hunk 3 above)
Comment (amirmehrkar) [reply snippets/terms.md:30]: yes

**Review by ghickman — APPROVED** [review]

**Review by ghickman — APPROVED** [review]

**Pull request merged**